src/main/kotlin/com/manimdsl/linearrepresentation/LinearRepresentation.kt

```diff
@@ -44,6 +46,17 @@ data class MoveObject(
     }
 }
 
+data class RestyleObject(
+    val shape: Shape,
+    val newStyle: StylesheetProperty,
+): ManimInstr {
+    override fun toPython(): List<String> {
+        return if (shape is StyleableShape) {
```

Comment:
Can this function not just accept StyleableShape the type for the shape parameter?

Reply:
There's a problem from the AST executor that if I do the check it won't do the cast, I decided it was nicer to do the check inside this instead of a let everywhere